scripts/prepublish.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const regex = /module\.exports = punycode;/;
+
+var sourceContents = fs.readFileSync(path.resolve(__dirname, '../punycode.js'), 'utf-8');
+
+if (!regex.test(sourceContents)) {
```

Comment:
Opposed to includes for Node.js compat reasons: http://kangax.github.io/compat-table/es6/#test-String.prototype_methods_String.prototype.includes

Your call?